interfaces/redis/interface.go

```diff
@@ -0,0 +1,10 @@
+package redis
+
+import "github.com/go-redis/redis/v8"
+
+type Redis interface {
+	GetClient() *redis.Client
```

Comment:
Do we actually need to expose this logic to consumers? It can be hidden in the plugin. What are the benefits?

Reply:
Other plugins receive an interface, which provides an underline client.